criu/crtools.c

```diff
@@ -185,7 +222,7 @@ int main(int argc, char *argv[], char *envp[])
 	}
 
 	/* We must not open imgs dir, if service is called */
-	if (strcmp(argv[optind], "service")) {
+	if (opts.mode != CR_SERVICE) {
```

Comment:
Could you please add something like the following to fix a segmentation fault for `criu cpuinfo`?
```C
if (opts.mode == CR_CPUINFO && argc < 3) {
        pr_err("cpuinfo requires an action: dump or check\n");
        goto usage;
}
```

Reply:
Ok, will do.